docs/concept-workflows.md

```diff
@@ -1,30 +1,210 @@
 ---
 id: concept-workflows
-title: Workflows
+title: The concept of Workflows
```

Comment:
do we have any other concept docs that start out named "The concept of"? why this one?

Reply:
Because this is the first one I have re-written :)